vitrina/resources/migrations/0007_datasetdistribution_status.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 3.2.25 on 2025-08-26 14:06
+
+from django.db import migrations, models
+import django.db.models.deletion
+import vitrina.resources.models
+
+
+def populate_existing_rows(apps, schema_editor):
+    DatasetDistribution = apps.get_model("vitrina_resources", "DatasetDistribution")
+    Concept = apps.get_model("vitrina_classifiers", "Concept")
+
+    completed = Concept.objects.get(code="COMPLETED")
+    develop = Concept.objects.get(code="DEVELOP")
+
+    for distribution in DatasetDistribution.objects.all():
+        if distribution.dataset.access_rights == "PUBLIC":
+            distribution.status = completed
+        else:
+            distribution.status = develop
+        distribution.save()
```

Comment:
Update queries should be faster, I think. But this won't trigger `.save(..)` any signals and won't update `auto_add` fields.
```suggestion
    DatasetDistribution.objects.filter(dataset__access_rights="PUBLIC").update(status=completed)
    DatasetDistribution.objects.exclude(dataset__access_rights="PUBLIC").update(status=develop)
```

Reply:
Changed, thanks